src/index.html

```diff
@@ -7,8 +7,76 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="https://www.moyo.ua/" class="img-box">
+        <img
+            class="logo"
+            src="images/logo.png"
+            alt="Moyo logo"
+          >
+      </a>
+      <nav class="navigation">
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="#"
+              class="link is-active">APPLE
+            </a>
+          </li>
```

Comment:
not changed from prev review

```suggestion
          <li class="nav-item">
            <a
              href="#"
              class="link is-active"
              >
                APPLE
            </a>
          </li>
```